addons/default/visiosoft/media-field_type/src/Table/ValueTableBuilder.php

```diff
@@ -66,7 +66,7 @@ class ValueTableBuilder extends TableBuilder
         ],
         'rotate' => [
             'target' => '_blank',
-            'icon' => 'fa fa-sync-alt',
+            'icon' => 'fa fa-sync-alt fa-refresh',
```

Comment:
Does it support multiple icons?

Reply:
dashboard and frontend work differently, this makes it appear on both.